inc/item_disk.class.php

```diff
@@ -464,6 +506,121 @@ public static function rawSearchOptionsToAdd($itemtype) {
          ]
       ];
 
+      $tab[] = [
+         'id'                 => '174',
+         'table'              => self::getTable(),
+         'field'              => 'encryption_status',
+         'name'               => __('Encryption status'),
+         'searchtype'         => 'equals',
+         'forcegroupby'       => true,
+         'massiveaction'      => false,
+         'searchequalsonfield' => true,
+         'datatype'           => 'specific',
+         'joinparams'         => [
+            'jointype'           => 'itemtype_item'
+         ]
+      ];
+
+      $tab[] = [
+         'id'                 => '175',
+         'table'              => self::getTable(),
+         'field'              => 'encryption_tool',
+         'name'               => __('Encryption tool'),
+         'forcegroupby'       => true,
+         'massiveaction'      => false,
+         'datatype'           => 'string',
+         'joinparams'         => [
+            'jointype'           => 'itemtype_item'
+         ]
+      ];
+
+      $tab[] = [
+         'id'                 => '176',
+         'table'              => self::getTable(),
+         'field'              => 'encryption_algorithm',
+         'name'               => __('Encryption algorithm'),
+         'forcegroupby'       => true,
+         'massiveaction'      => false,
+         'datatype'           => 'string',
+         'joinparams'         => [
+            'jointype'           => 'itemtype_item'
+         ]
+      ];
+
+      $tab[] = [
+         'id'                 => '177',
+         'table'              => self::getTable(),
+         'field'              => 'encryption_type',
+         'name'               => __('Encryption type'),
+         'forcegroupby'       => true,
+         'massiveaction'      => false,
+         'datatype'           => 'string',
+         'joinparams'         => [
+            'jointype'           => 'itemtype_item'
+         ]
+      ];
+
       return $tab;
    }
+
+   static function getEncryptionStatusDropdown($value = self::ENCRYPTION_STATUS_NO, $options = []) {
```

Comment:
`get` is not OK, this we do not return anything. Please also add phpdoc

Reply:
BTW, note that new UI will certainly require at some point to get possible status list (`$allStatus`) from a real `getSthing` ;)